isis/tests/FunctionalTestsApolloPanStitcher.cpp

```diff
@@ -0,0 +1,42 @@
+#include <QTemporaryDir>
+#include <memory>
+
+#include "apollopanstitcher.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+#include "Endian.h"
+#include "PixelType.h"
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+#include "Fixtures.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/apollopanstitcher.xml").expanded();
+
+TEST_F(TempTestingFiles, FunctionalTestsApolloPanStitcherDefault) {
+  QVector<QString> args = {"file_base=$ISISTESTDATA/isis/src/apollo/apps/apollopanstitcher/tsts/default/input/AS15_P_0177R10",
```

Comment:
Did we reduce this data?  Running an ls -lah shows that each file is ~39MB, and it looks like the output is being put in the ISISTESTDATA area, so we're increasing data usage by 234MB.

Reply:
The input `reduce8.cub` can be removed from the data area. We have to keep the eight ~39MB cubes and the new test output is being stored in the temp directory.

Reply:
Sorry for being annoying, but why can't they be reduced?  I guess I don't understand apollopanstitch well enough to understand why we can't downsample the cubes in the same way as other tests.

Reply:
The cubes are stitched together by "...find[ing] the centers of the [fiducial marks](https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/Apollo14Shadows.jpg/800px-Apollo14Shadows.jpg) along the image segments." I believe that the actual image DNs are searched for these fiducial marks and reducing or modifying the images could potentially break this test case. 

I brought this up last week during stand up and the consensus seemed to be to reduce the data footprint as easily as possible. The easiest thing to do is remove the stored output file (~240MB file) since we can test it in a gtest, so we reduce the data being used with minimal effort. I could look into reproducing the fiducial marks but that may take a while for a test case. I figured this was one of the few cases where using the full set of test data is worth the storage space.